ui/app/components/oidc/scope-form.js

```diff
@@ -24,17 +24,14 @@ export default class OidcScopeFormComponent extends Component {
 
   @tracked modelValidations;
 
-  exampleTemplate = JSON.stringify(
-    {
-      username: '{{identity.entity.aliases.$MOUNT_ACCESSOR.name}}',
-      contact: {
-        email: '{{identity.entity.metadata.email}}',
-        phone_number: '{{identity.entity.metadata.phone_number}}',
-      },
-    },
-    null,
-    2
-  );
+  exampleTemplate = `{
+  "username": {{identity.entity.aliases.$MOUNT_ACCESSOR.name}},
+  "contact": {
+    "email": {{identity.entity.metadata.email}},
+    "phone_number": {{identity.entity.metadata.phone_number}}
+  },
+  "groups": {{identity.entity.groups.names}}
+}`;
```

Comment:
I had originally assumed that the value passed into the `JsonEditor` would need to be properly formatted json. I tried creating a scope and copied the example template but the backend threw an error regarding the template formatting. I updated this to a template literal and changed the `JsonEditor` mode to `ruby` and now it is displaying correctly in code mirror and formatted correctly for the backend.

Reply:
I'm actually going to add a comment in the code in regards to the spacing just in case anyone is tempted to update it in the future.

Reply:
Good idea!